lesson7..html

```diff
@@ -5,76 +5,181 @@
-} else {
-    //hide btn
-document.getElementById('button')./*style.display='none'*/disabled = true; //an
-}
-}
+        <script>
+            var data = [
+                {
+                    Id: 112,
+                    Name: 'Nguyen Van B',
+                    Age: 18,
+                },
+                {
+                    Id: 234,
+                    Name: 'Nguyen Van D',
+                    Age: 14,
+                },
+                {
+                    Id: 245,
+                    Name: 'Nguyen Van A',
+                    Age: 53,
+                },
+                {
+                    Id: 345,
+                    Name: 'Nguyen Van C',
+                    Age: 34,
+                }
+            ]
+            function showInfodata(abc) {
+                var hang = "";
+                for (x in data) {
+                    hang += `<tr>
+                  <td>${data[x]["Id"]}</td>
+                  <td>${data[x]["Name"]}</td>
+                  <td>${data[x]["Age"]}</td>
+              </tr>`;
+                }
+                document.getElementById("body").innerHTML = hang;
+            }
+            showInfodata(data);
+            //show list data
+
+            function add() {
+                var item_id = document.getElementById('id').value //gán đưa đến phần tử cần xử lý thêm value
+                var item_name = document.getElementById('name').value
+                var item_age = document.getElementById('age').value
 
+                var item = {
+                    Id: item_id,
+                    Name: item_name,
+                    Age: item_age
+                }
+                data.push(item) //đưa dữ liệu lên data
+                render()
+                clead()
+            }
 
-    </script>
+            function render() { //kết xuất
+                table = `  <tr>
+            <th>ID</th>
+            <th>Name</th>
+            <th>Age</th>
+        </tr>`
+                for (let i = 0; i < data.length; i++) {
+                    table += `  <tr>
+            <td>${data[i].Id}</td>
+            <td>${data[i].Name}</td>
+            <td>${data[i].Age}</td>
+        </tr>`
+                }
+                document.getElementById('render').innerHTML = table
+            }
+            function clead() {
+                document.getElementById('id').value = "";
+                document.getElementById('name').value = "";
+                document.getElementById('age').value = "";
+            } //khóa lại sau mỗi lần nhập lấy giá trị rỗng 
+      //order
+            function orderName() {
+                data.sort(function (a, b) {
+                    let x = a.Name.toLowerCase();
+                    let y = b.Name.toLowerCase();
+                    if (x < y) { return -1; }
+                    if (x > y) { return 1; }
+                    return 0;
+                });
+                add();
```

Comment:
```suggestion
                
                showDataInfo(data)
```
ở đây cũng thế chỉ cần gọi lại thế này thôi